benchmarks/data-serving/server/Dockerfile

```diff
@@ -1,11 +1,11 @@
-FROM cloudsuite/cassandra:3.11.3
+FROM cloudsuite/cassandra:4.0
 
 COPY files /scripts
 
 RUN chmod +x /scripts/docker-entrypoint.sh
 
 ENTRYPOINT ["/scripts/docker-entrypoint.sh"]
 
-CMD ["cassandra", "-R", "-f"]
+CMD ["/apache-cassandra-4.0-alpha3/bin/cassandra", "-R", "-f"]
```

Comment:
this should not be an absolute hardcoded path; should use the defined ENV.

Reply:
This hasn't been resolved yet!

Reply:
Updated